client/settings/state/values/actions.js

```diff
@@ -62,7 +63,16 @@ export const submit = ( schema, silent ) => ( dispatch, getState, { callbackURL,
 		}
 	};
 
-	const coercedValues = coerceFormValues( schema, getState().form.values );
+	const coercedValues = coerceFormValues( formSchema, getState().form.values );
```

Comment:
Fixed in https://github.com/Automattic/woocommerce-services/pull/809/commits/57e2960ea57ae406c27a4469a2d820caf221073e.